src/Hazelcast.Net/CP/CPSubsystem.cs

```diff
@@ -31,6 +34,8 @@ internal class CPSubsystem : ICPSubsystem, IAsyncDisposable
         private readonly Cluster _cluster;
         private readonly SerializationService _serializationService;
         private readonly ConcurrentDictionary<string, IFencedLock> _fencedLocks = new ConcurrentDictionary<string, IFencedLock>();
+        private readonly DistributedObjectFactory _objectFactory;
```

Comment:
Is this used, or am I missing something?

Reply:
Good catch! Removed at 1e035072a014026fe3dd6f8b5a3f8970f4faf5a2